QuantumESPRESSO/run.sh

```diff
@@ -0,0 +1,204 @@
+if [[ $EESSI_CVMFS_REPO == "/cvmfs/software.eessi.io" ]] && [[ $EESSI_VERSION == "2023.06" ]]; then
+    echo "Running demo for QuantumESPRESSO 7.3.1 ..."
+    module load QuantumESPRESSO/7.3.1-foss-2023a
+    module load gnuplot/5.4.8-GCCcore-12.3.0
+else
+    echo "Don't know which QuantumESPRESSO module to load for ${EESSI_CVMFS_REPO}/versions/${EESSI_VERSION}" >&2
+    exit 1
+fi
+
+export OMP_NUM_THREADS=1
+export OMPI_MCA_osc=^ucx
+export OMPI_MCA_btl=^openib,ofi
+export OMPI_MCA_pml=^ucx
+export OMPI_MCA_mtl=^ofi
```

Comment:
We need a better solution for this in EESSI (although it is really an OpenMPI bug). Approach is fine here to dance around it.